build-common/src/org/jetbrains/kotlin/incremental/buildUtil.kt

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
As far as I understand these functions are copies of functions at KotlinBuilder. Why not to reuse them at KotlinBuilder? I mean, these functions are in build common, so their originals from KotlinBuilder can be removed.